### PR TITLE
PLAT-37974 freeze corretto 16.0.2.7.1

### PR DESCRIPTION
**Pull request by c3-YukiK**

### CODE REVIEW

File: Casks/corretto16.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask "corretto" do
```
Comment (HarryWeppner) [inline Casks/corretto16.rb:1]: You want to name this corretto16

**Review by HarryWeppner — CHANGES_REQUESTED** [review]
Pick a specific name

**Comment by c3-YukiK** [comment]
@HarryWeppner made the change

**Review by HarryWeppner — APPROVED** [review]
Lgtm

**Pull request merged**